docs/patterns/single-new-eks-observability-accelerators/single-new-eks-mixed-observability.md

```diff
@@ -0,0 +1,132 @@
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [cdk](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install)
+4. [npm](https://docs.npmjs.com/cli/v8/commands/npm-install)
+
+## Deploying
+
+1. Clone your forked repository
+
+```sh
+git clone https://github.com/aws-observability/cdk-aws-observability-accelerator.git
+```
+
+2. Install the AWS CDK Toolkit globally on your machine using
+
+```bash
+npm install -g aws-cdk
+```
+
+3. Install project dependencies by running `npm install` in the main folder of this cloned repository
+
+4. Once all pre-requisites are set you are ready to deploy the pipeline. Run the following command from the root of this repository to deploy the pipeline stack:
+
+```bash
+make build
+make pattern single-new-eks-native-observability deploy
+```
+
+## Verify the resources
+
+Run update-kubeconfig command. You should be able to get the command from CDK output message.
+
+```bash
+aws eks update-kubeconfig --name single-new-eks-mixed-observability-accelerator --region <your region> --role-arn arn:aws:iam::xxxxxxxxx:role/single-new-eks-opensource-singleneweksopensourceob-82N8N3BMJYYI
+```
+
+Let’s verify the resources created by steps above.
+
+```bash
+kubectl get nodes -o wide
+```
+
+Output:
+
+```console
+NAME                                         STATUS   ROLES    AGE    VERSION               INTERNAL-IP    EXTERNAL-IP   OS-IMAGE         KERNEL-VERSION                  CONTAINER-RUNTIME
+ip-10-0-144-134.us-west-1.compute.internal   Ready    <none>   143m   v1.25.9-eks-0a21954   10.0.144.134   <none>        Amazon Linux 2   5.10.179-168.710.amzn2.x86_64   containerd://1.6.19
+```
+
+Next, lets verify the namespaces in the cluster:
+
+```bash
+kubectl get ns # Output shows all namespace
+```
+
+Output:
+
+```console
+NAME                            STATUS   AGE
+aws-for-fluent-bit              Active   142m
+cert-manager                    Active   142m
+default                         Active   148m
+external-secrets                Active   142m
+kube-node-lease                 Active   149m
+kube-public                     Active   149m
+kube-system                     Active   149m
+opentelemetry-operator-system   Active   142m
+prometheus-node-exporter        Active   142m
+```
+
+## Visualization
+
+Navigate to CloudWatch and go to Metrics -> All Metrics.
+
+Select the metrics in the ContainerInsights/Prometheus Namespace:
+
+![metric selectrion](../images/mixed-metrics.png)
+
+View the graph of the selected metrics:
+
+![metric graph](../images/mixed-graph.png)
+
+## Viewing Logs
+
+By default, we deploy a FluentBit daemon set in the cluster to collect worker logs for all namespaces. Logs are collected and exported to Amazon CloudWatch Logs, which enables you to centralize the logs from all of your systems, applications,
+and AWS services that you use, in a single, highly scalable service.
+
```

Comment:
Could we show anything from XRay Console like Map. If nothing i understand its because of no workloads